code/clue.js

```diff
@@ -1,59 +1,334 @@
+}
+
+const billiardRoom = {
+  name: 'Billiard Room',
+  image: 'assets/billiardroom.png'
+}
+
+const lounge = {
+  name: 'Lounge',
+  image: 'assets/lounge.png'
+}
+
+const ballroom = {
+  name: 'Ballroom',
+  image: 'assets/ballroom.png'
+}
+
+const hall = {
+  name: 'Hall',
+  image: 'assets/hall.png'
+}
+
+const spa = {
+  name: 'Spa',
+  image: 'assets/spa.png'
+}
+
+const livingRoom = {
+  name: 'Living Room',
+  image: 'assets/livingroom.png'
+}
+
+const observatory = {
+  name: 'Observatory',
+  image: 'assets/observatory.png'
+}
+
+const theater = {
+  name: 'Theater',
+  image: 'assets/theater.png'
+}
+
+const guestHouse = {
+  name: 'Guest House',
+  image: 'assets/guesthouse.png'
+}
+
+const patio = {
+  name: 'Patio',
+  image: 'assets/patio.png'
+}
+
+// SUSPECTS, WEAPONS AND ROOMS GROUPED IN ARRAYS:
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
+  mrsWhite,
+]
+
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
 ]
 
-const weapons = []
+const rooms = [
+  diningRoom,
+  conservatory,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  lounge,
+  ballroom,
+  hall,
+  spa,
+  livingRoom,
+  observatory,
+  theater,
+  guestHouse,
+  patio,
+]
 
-const rooms = []
+// FUNCTION RANDOMLY SELECTING ONE ITEM FROM THE ABOVE ARRAYS:
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+// OBJECT KEEPING MYSTERY:
+
+const mystery = {
+  killer: '',
+  weapon: '',
+  room: '',
+}
+
+// FUNCTION INVOKED WHEN YOU CLICK ON THE KILLER CARD, CLICK AGAIN TO RESET:
 
-// This function will be invoked when you click on the killer card.
 const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
-  mystery.killer = randomSelector(suspects)
+  if (document.getElementById('killerCard').classList.contains('shown')) {
+    mystery.killer = ''
 
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
-  document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+    document.getElementById('killerCard').style.background = '#f7876b'
+    document.getElementById('killerName').innerHTML = ''
+    document.getElementById('killerImage')
+    killerImage.src = ''
+
+    killerCard.classList.remove('shown')
```

Comment:
I don't know if "shown" is a great naming for this. Shown means that it have already been showed instead of it suppost to show
```suggestion
    killerCard.classList.remove('show')
```